src/ngAnimate/animateQueue.js

```diff
@@ -583,7 +586,7 @@ var $$AnimateQueueProvider = ['$animateProvider', function($animateProvider) {
      * d) the element is not a child of the $rootElement
      */
     function areAnimationsAllowed(element, parentElement, event) {
-      var bodyElement = jqLite($document[0].body);
+      var bodyElement = jqLite(rawDocument.body);
```

Comment:
No, in [animateCssDriver.js][1].

[1]: https://github.com/Narretz/angular.js/blob/d9c1d330e35af76f1de7ecb59adbf317769b43e1/src/ngAnimate/animateCssDriver.js#L22